bots/elasticsearch/bot.py

```diff
@@ -150,6 +257,13 @@ def setup(self):
         logger.debug('Waiting for yellow status')
         self.es_client.cluster.health(wait_for_status='yellow')
 
+        logger.info('Putting Elasticsearch settings')
```

Comment:
Can this setup be in an `if` block that defaults to false?  That way the existing indexing can be kept alive and well without have to change the jobs or anything. Setting up the new one will just require an extra flag to be set.